src/main/scala/com/gu/elasticsearchmonitor/ClusterHealth.scala

```diff
@@ -58,7 +58,7 @@ object ClusterHealth {
       case Success(response) =>
         Left(s"Unable to fetch the cluster health status. Http code ${response.code}")
 
-      case Failure(NonFatal(e)) =>
+      case Failure(e) =>
```

Comment:
@jacobwinch Could you confirm you change from `Failure(NonFatal(e))` to `Failure(e)` because of the exception raised was a fatal one, then there will be no match, and so the match was non exhaustive?

Reply:
That's correct; I was receiving compiler warnings related to non-exhaustive matches and I wanted to tidy them up so that I didn't miss any new warnings/errors